linter/rules/model_file_should_not_have_wildcard_view_includes.py

```diff
@@ -0,0 +1,17 @@
+from linter.rule import Rule
+from typing import Any, Tuple
+from re import search
+
+
+class ModelFileShouldNotHaveWildcardViewIncludes(Rule):
```

Comment:
Is there any reason we should allow *.dashboard and *.explore wildcard includes as well? Do we not run into the same compilation speed issue?  I get that generally there are going to be far more view files than others, but it seems odd to enforce this so selectively.  If people are going to go through the trouble of explicitly including every view they can also do it for the lower number of dashboards and explores.  This is debatable, of course, so I'm curious about your perspective.

One thing though, technically the wildcard view includes can happen from any file, like an explore file, so semantically I think this rule should just be "IncludeShouldNotHaveWildcard" (or "InludeShouldNotHaveViewsWithWildcard" depending on where we land with this), especially as we incorporate the suggestion above about treating "includes" as their own object.

Finally, the regex `\*.view` should be `\*\.view`, as the current version will match an asterisk, followed by _any character_ (instead of just a period), followed by "view".

Reply:
Based on our conversation earlier, I updated the class name to "IncludeShouldNotHaveViewsWithWildcard".